src/Swarm/Language/Capability.hs

```diff
@@ -304,7 +305,6 @@ constCaps =
     Appear -> [CAppear] -- paint?
     Whereami -> [CSenseloc] -- GPS?
     Random -> [CRandom] -- randomness device (with bitcoins)?
-    Whoami -> [CWhoami] -- mirror, needs a recipe
```

Comment:
I think we might have had a similar idea, so I caused you a merge conflict. Sorry about that :sweat_smile:

Reply:
Hah, no worries.